### PR TITLE
index methods and testing for Task

### PR DESCRIPTION
**Pull request by tabithatrubey**

### CODE REVIEW

**Comment by renitagid** [comment]
FYI, we accidentally named the branch in correctly, this is just for the index method!

File: spec/requests/tasks_spec.rb [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 RSpec.describe "Tasks", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of tasks" do
+      user = User.create(name: "Renita", email: "hello@world.com", password: "1234hi")
```
Comment (elysemontano) [inline spec/requests/tasks_spec.rb:6]: Make sure that your user includes all attributes needed to create a user.  Down the line this test will fail when validations are added in.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Make sure that your user includes all attributes needed to create a user. Down the line this test will fail when validations are added in.

File: spec/requests/tasks_spec.rb (hunk 1 above)
Comment (renitagid) [reply spec/requests/tasks_spec.rb:6]: So, just to clarify - we already did do our validations. We are intentionally making only the name be required, not the profile picture or bio. Is there something I'm missing?

**Review by elysemontano — APPROVED** [review]

**Pull request merged**